backends/rapidpro/task.go

```diff
@@ -32,15 +29,18 @@ func queueMsgHandling(rc redis.Conn, c *Contact, m *Msg) error {
 
 func queueChannelEvent(rc redis.Conn, c *Contact, e *ChannelEvent) error {
 	body := map[string]any{
-		"org_id":      e.OrgID_,
-		"contact_id":  e.ContactID_,
+		"event_id":    e.ID_,
+		"event_type":  e.EventType_,
 		"urn_id":      e.ContactURNID_,
 		"channel_id":  e.ChannelID_,
 		"extra":       e.Extra(),
 		"new_contact": c.IsNew_,
 		"occurred_on": e.OccurredOn_,
 		"created_on":  e.CreatedOn_,
 	}
+	if e.OptInID_ != 0 {
+		body["optin_id"] = e.OptInID_
```

Comment:
best I can tell we're not doing optin triggers correctly in the engine because this isn't being passed - the triggers work but `@optin` is exposed in expressions